yo/app/scripts/model.js

```diff
@@ -0,0 +1,50 @@
+/*jshint undef: false, unused: false, indent: 2*/
```

Comment:
What exactly is the use of this file? Could you document? It seems to me it shoult not be in the root scripts folder, but I can't be sure if I don't know what it does.